WebDriverAgentLib/Utilities/FBFailureProofTestCase.m

```diff
@@ -28,6 +22,25 @@ - (void)setUp
   self.shouldHaltWhenReceivesControl = NO;
 }
 
+- (void)_recordIssue:(XCTIssue *)issue
+{
+  NSString *description = [NSString stringWithFormat:@"%@ (%@)", issue.compactDescription, issue.associatedError.description];
+  [FBLogger logFmt:@"Issue type: %ld", issue.type];
+  [self _enqueueFailureWithDescription:description
+                                inFile:issue.sourceCodeContext.location.fileURL.path
+                                atLine:issue.sourceCodeContext.location.lineNumber
+                              expected:issue.type == 5];
```

Comment:
Perhaps we should add a comment here regarding the meaning of 5 (`XCTIssueTypeUnmatchedExpectedFailure`)?

Reply:
added